src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -209,23 +199,7 @@ public void Initialize()
         {
             lock (_syncLock)
             {
-                if (InStandbyMode)
-                {
-                    if (!_warmupComplete)
-                    {
-                        if (!_webHostSettings.IsSelfHost)
-                        {
-                            HostingEnvironment.QueueBackgroundWorkItem((ct) => WarmUp(_webHostSettings, EventManager));
-                        }
-                        else
-                        {
-                            Task.Run(() => WarmUp(_webHostSettings, EventManager));
```

Comment:
In placeholder mode we used to only execute this warmup once on startup, and spin up an empty host. Now we spin up a host with the single warmup function that can be pinged as necessary.